refinery/tool_manager/utils.py

```diff
@@ -8,13 +8,21 @@
 from jsonschema import RefResolver, validate, ValidationError
 
 from factory_boy.django_model_factories import (FileRelationshipFactory,
+                                                GalaxyParameterFactory,
                                                 InputFileFactory,
                                                 OutputFileFactory,
+                                                ParameterFactory,
                                                 ToolDefinitionFactory)
 
 from file_store.models import FileType
+from .models import ToolDefinition
 
 logger = logging.getLogger(__name__)
+ANNOTATION_ERROR_MESSAGE = (
```

Comment:
I think it's [best to avoid "one log statement != one written log"](http://stackoverflow.com/questions/4669590/displaying-a-log-per-line-for-a-multiline-text)